pom.xml

```diff
@@ -532,12 +532,22 @@
                                         <exclude>javax.servlet:javax.servlet-api</exclude>
                                         <!-- Replaced with jakarta.validation:jakarta.validation-api -->
                                         <exclude>javax.validation:validation-api</exclude>
-                                        <!-- Replaced with jakarta.xml.bind:jakarta.xml.bind-api -->
-                                        <exclude>javax.xml.bind:jaxb-api</exclude>
+                                        <!-- Replaced with jakarta.xml.bind:jakarta.xml.bind-api, but need to keep around for old plugins -->
+                                        <!-- <exclude>javax.xml.bind:jaxb-api</exclude> -->
```

Comment:
Is this commented out exclusion needed?

Reply:
Kinda. 
This is a result of  

> May or may not create issue to update file provisioning plugins. Having the old jaxb around seems enough to get them to work, don't know if that will eventually cause issues (although they are simpler than swagger/openapi client libraries)

Ideally, we'd update the plugins and then add this ban. Allowing this library and the new jarkarta ones at the same time seems to work right now, but may lead to problems in the future. 
In other words, this could be considered aspirational.